packages/web/src/styles.ts

```diff
@@ -24,25 +23,37 @@ export enum colors {
   purple = '#BF97FF',
   deepBlue = '#3488EC',
   lightBlue = '#73DDFF',
+  cyan = '#3DBFFF',
   turquoise = '#29EDFF',
 
   greenScreen = '#0CDA6E',
   purpleScreen = '#924EFF',
   redScreen = '#FF6553',
   blueScreen = '#52B6FF',
+
+  // added
+  faintGray = '#F8F9F9',
+  faintPurple = '#EFE5FF',
+  faintGold = '#FEF2D6',
+  faintRed = '#FEDEDA',
+  faintCyan = '#DCF6FF',
+  lightGray = '#EDEEEF',
+
+  grayHeavy = '#ABADAF',
+  black = '#000',
 }
 
 export enum typeFaces {
-  futura = 'futura-pt, sans-serif',
-  garamond = 'eb-garamond, serif',
+  futura = 'Jost, futura-pt, futura, sans-serif',
+  garamond = 'EB Garamond, eb-garamond, Garamond, serif',
 }
 
-export const fonts = StyleSheet.create({
```

Comment:
change this so that on typography page i could print the styles out and know for sure that they matched. DRY